code/upaya/app/assets/stylesheets/main.scss

```diff
@@ -1,2 +1,2 @@
-@import "us_web_design_standards";
-@import "us_web_design_standards_fonts";
\ No newline at end of file
+/*@import "us_web_design_standards";
+@import "us_web_design_standards_fonts";*/
```

Comment:
These should be `require` statements in our application.css
https://github.com/18F/us_web_design_standards_gem#rails